src/Symfony/Component/Form/FormErrorIterator.php

```diff
@@ -265,6 +266,16 @@ public function seek($position)
         }
     }
 
+    public function getFormErrorByCode($code)
```

Comment:
`FormErrorIterator::getFormErrorsByCode` duplicates terms, the context is clear due the class name. We're dealing with form errors. Ie. you could go with `FormErrorIterator::getByCode` or maybe `findByCode`.

However if we can get a _new_ `FormErrorIterator` instance for one _or more_ codes that would be truely convenient.. ie.
`$specificErrros = $errors->withCodes(array(Constraint::CODE, OtherConstraint::CODE));`